en/03/03.adoc

```diff
@@ -48,3 +48,15 @@ refractometer readings. You must always enter the current Plato and either the
 original Plato or the OG. If you just want to find out what specific gravity
 11.2 Plato corresponds to, enter 11.2 in both the original Plato and current
 Plato fields.
+
+=== Hydrometer Temp Adjustmen
+
+This tool adjust the gravity reading of your hydrometer based on temperature.
+
+Tools-{gt}"Hydrometer Adjustment Tool".  Input the gravity reading (SG Reading) from your hydrometer and the temperature and click "convert".  The "Adjust SG" will be populated with the calculation.  If needed, change the value of the "Hydrometer Calibration" to match your hydrometer.
```

Comment:
"will be populated with the calculation" change "the calculation" by what the result of this calculation represents (ex. a temperature adjusted SG)